docs/fides/docs/guides/generate_resources.md

```diff
@@ -134,6 +134,22 @@ These can be supplied in an IAM policy:
 }
 ```
 
+## Filtering AWS Resources
+
+It is possible to filter resources at the organization level by adding a resource filter within `fidesctl_meta`. The `ignore_resource_arn` filter can exclude any resources with an exact matching arn or also supports wildcarding individual arn fields. An empty arn field in the filter pattern works as a wildcard.
```

Comment:
```suggestion
It is possible to filter resources at the organization level by adding a resource filter within `fidesctl_meta`. The `ignore_resource_arn` filter can exclude any resources with an exact matching Amazon Resource Name (ARN) and also supports wildcards in individual ARN fields. An empty ARN field in the filter pattern works as a wildcard.
```

But, `An empty ARN field in the filter pattern works as a wildcard.` seems confusing. Can you add some detail to demonstrate exactly how wildcards look, and how they're handled? Another example would be nice as well.

Reply:
sure I'll add more examples